src/Domain/DomainHelpers.hpp

```diff
@@ -108,13 +108,20 @@ wedge_coordinate_maps(double inner_radius, double outer_radius,
 
 /// \ingroup ComputationalDomainGroup
 /// These are the ten Frustums used in the DomainCreators for binary compact
-/// objects. The cubes enveloping the two Shells each have a side length of
-/// `length_inner_cube`. The ten frustums also make up a cube of their own,
-/// of side length `length_outer_cube`.
+/// objects. The Frustums partition the volume defined by two bounding
+/// surfaces: The inner surface is the surface of the two joined inner cubes
+/// enveloping the two compact objects, while the outer is the surface of the
+/// outer cube. The cubes enveloping the two Shells each have a side length of
+/// `length_inner_cube`. The outer cube has a side length of
+/// `length_outer_cube`. `origin_preimage` is a parameter
```

Comment:
What map is this the preimage of the origin under?  I.e., why is the argument named what it is?

Reply:
The two cubes in the BBH domain by default have their middle point centered at the origin. A translation map is then applied that moves the two cubes away from the origin. This is done to move the COM of the objects+waves to the origin. The `origin_preimage` is then the location of the COM before it is translated to the origin.

The frustums need to know where the two center cubes are placed so that they can wrap around them properly.
Let me know if I can clarify either my comment or documentation :+1: